js/wp-seo-metabox-category.js

```diff
@@ -0,0 +1,217 @@
+/* global wp, _, wpseoPrimaryCategoryL10n */
+(function( $ ) {
+	'use strict';
+
+	var primaryTermInputTemplate, primaryTermUITemplate, primaryTermScreenReaderTemplate;
+	var taxonomies = wpseoPrimaryCategoryL10n.taxonomies;
+
+	/**
+	 * Checks if the elements to make a term the primary term and the display for a primary term exist
+	 *
+	 * @param checkbox
+	 *
+	 * @return {bool}
+	 */
+	function hasPrimaryTermElements( checkbox ) {
+		return 1 === $( checkbox ).closest( 'li' ).children( '.wpseo-make-primary-term' ).length;
+	}
+
+	/**
+	 * Retrieves the primary term for a taxonomy
+	 *
+	 * @param taxonomyName
+	 * @returns {string}
```

Comment:
According to [JSDoc](http://usejsdoc.org/tags-returns.html) both may be used. For consistency I changed it to `@returns`, also because PHPStorm automatically generates this.